agent/src/com/jonnyzzz/teamcity/virtual/VMDetector.java

```diff
@@ -104,7 +129,7 @@ private String executeCommandWithShell(@NotNull final String name, @NotNull fina
           return null;
         }
 
-        return result.getStdout();
+        return result.getOutLines();
```

Comment:
Not relevant after this [commit](https://github.com/jonnyzzz/TeamCity.Virtual/pull/51/commits/7da68b403fe776c8f829420c75889f9981a73f65)